package.json

```diff
@@ -37,10 +37,11 @@
     "test": "tap --timeout 300 test/test-*.js"
   },
   "dependencies": {
-    "async": "^1.0.0",
+    "async": "^2.0.0",
+    "byline": "^5.0.0",
```

Comment:
Looks like you did `--save` instead of `--save-dev` when you upgraded byline. It is still in `devDependencies` as `^4.2.1`.